src/types/vec/vec.zig

```diff
@@ -698,9 +698,10 @@ fn SignedNumericVec3Mixin(comptime Self: type, comptime Dimens: comptime_int, co
             /// Modifies vector itself so that it is a cross product of vectors, i.e. a vector that is perpendicular to the plane
             /// containing x and y and has a magnitude that is equal to the area of the parallelogram that x and y span.
             pub fn crossSelf(a: *Self, b: Self) void {
```

Comment:
It would be good to add tests for crossSelf function. Since this library is heavy WIP a lot of tests are missing. Add tests for crossSelf function. Use my tests as example, be consistent.